src/rpc/client/ts/src/server.ts

```diff
@@ -80,22 +56,52 @@ export async function startServer(
     rootPath: rootPath,
     port: port,
   })
-  const artifact = new Artifact('omega-edit-grpc-server', version, rootPath)
 
-  let args = [`--interface=${host}`, `--port=${port}`]
+  const scriptDir = path.join(rootPath, 'node_modules/omega-edit/bin')
```

Comment:
This doesn't appear to be used.  Line 80 has the goods.

Reply:
Its used on line 73 for a debug statement

Reply:
But then `--script-dir=${__dirname}` is on line 80.